src/Compilers/Test/Core/Platform/CoreClr/AssemblyLoadContextUtils.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if NETCOREAPP
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.Loader;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Roslyn.Test.Utilities.CoreClr
+{
+    public static class AssemblyLoadContextUtils
+    {
+        public static SimpleAssemblyLoadContext Create(string name, string? probingPath = null) =>
+            new SimpleAssemblyLoadContext(name, probingPath);
+    }
+
+    public sealed class SimpleAssemblyLoadContext : AssemblyLoadContext
+    {
+        private readonly string _probingPath;
+
+        public SimpleAssemblyLoadContext(string name, string? probingPath = null)
```

Comment:
It looks like `probingPath` is always `null` (couldn't find a caller that passes an actual value). Or did I miss something?

Reply:
It is `null` in our current use. This is a basic utility class I copied over though which supports alternate probing paths. It's a common helper for `AssemblyLaodContext` usage. Didn't want to delete it only to have to add it back later when I needed it.